frontend/src/components/eMIB/SideNavigation.jsx

```diff
@@ -49,7 +49,7 @@ class SideNavigation extends Component {
             </Nav>
           </Col>
           <Col sm={9}>
-            <Tab.Content>
+            <Tab.Content tabIndex={0}>
```

Comment:
Are you sure that it's the right syntax? I mean, it seems that everybody uses `tabindex="0"`

Reply:
In React this is what we had for all our previous components.